README.md

```diff
@@ -349,7 +349,7 @@ You can have more information on how to use Docker to build and try Dyna&omega;o
 
 <a name="documentation"></a>
 ## Dyna&omega;o Documentation
-You can download Dyna&omega;o documentation [here](https://github.com/dynawo/dynawo/releases/download/v1.4.1/DynawoDocumentation.zip).
+You can download Dyna&omega;o documentation [here](https://github.com/dynawo/dynawo/releases/download/nightly/DynawoDocumentation.pdf).
```

Comment:
I wuld propose 'nightly documentation'

Reply:
we don't want to keep the two links ?

Reply:
If I'm not mistaken we have one link at the beginning which is the release one, and this one in the middle if forthe nightly, maybe we could reorganize a bit

Reply:
nope you're right the is the only link available so let's provide both